packages/mobile/src/app/verificationMigration.ts

```diff
@@ -26,9 +23,20 @@ function* isAddressVerified(address: string | null) {
       TAG,
       `No number or pepper found - Number present? ${!!e164Number} - Pepper present? ${!!pepper}`
     )
+    return null
+  }
+  return getPhoneHash(e164Number, pepper)
+}
+
+function* isAddressVerified(address: string | null) {
```

Comment:
The diff is a bit annoying to read. I started doing more changes but ended up reverting all of them, in the end all I did is moving some stuff to separate functions: `fetchPhoneIdentifier` and `fetchMtwAddress`. No logic should have changed

Reply:
If anyone has any concerns with this I'm happy to revert, but I do think after the changes the file is better organized, it's just that this diff is a bit annoying to read.